lib/redux/actions/async.js

```diff
@@ -422,105 +422,134 @@ export function doVersionCheck() {
  * COMPLEX ACTION CREATORS
  */
 
-export function putTargetsInStorage() {
+export function setTargetTimezone(userId, timezoneName) {
```

Comment:
Move the `COMPLEX ACTION CREATORS` section header below all asynchronous action creators please, or if you don't think the division (between async action creators involving API calls and thunks that are just conditionally doing things or having synchronous side effects) is useful, get rid of the sections.